app/models/application_record.rb

```diff
@@ -1,4 +1,14 @@
 class ApplicationRecord < ActiveRecord::Base
   primary_abstract_class
   include HalApi::RepresentedModel
+
+  @@error_message_aliases = {}
```

Comment:
As part of the whole `ad_breaks` thing, I needed a way to proxy 1 fields error messages onto a different field in the UI.